sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -522,6 +522,15 @@ object SQLConf {
       .checkValue(_ >= 0, "The non-empty partition ratio must be positive number.")
       .createWithDefault(0.2)
 
+  val ADAPTIVE_OPTIMIZER_EXCLUDED_RULES =
+   buildConf("spark.sql.adaptive.optimizer.excludedRules")
```

Comment:
Since the PR description says nothing about adding this, I think its better to add it or separating this from this PR. The other part looks okay.